MAUI/CircularProgressBar/getting-started.md

```diff
@@ -11,6 +11,11 @@ documentation: ug
 
 This section explains the steps required to add the circular progress bar control with the progress and its customizable elements such as indeterminate, segment, progress fill, and track fill. This section covers only the basic features needed to know and gets started with the Syncfusion circular progress bar.
 
+To get start quickly with our .NET MAUI Circular ProgressBar, you can check the below video.
```

Comment:
To get start quickly with our .NET MAUI Circular ProgressBar, check the following video.

Reply:
Content modified.